src/mediatype-icons.ts

```diff
@@ -0,0 +1,39 @@
+import { html, LitElement } from 'lit';
+import { customElement, property } from 'lit/decorators.js';
+
+import { audioIcon } from './assets/img/icons/mediatype/audio';
+import { etreeIcon } from './assets/img/icons/mediatype/etree';
+import { imagesIcon } from './assets/img/icons/mediatype/images';
+import { filmIcon } from './assets/img/icons/mediatype/film';
+import { softwareIcon } from './assets/img/icons/mediatype/software';
+import { textsIcon } from './assets/img/icons/mediatype/texts';
+import { tvIcon } from './assets/img/icons/mediatype/tv';
+import { videoIcon } from './assets/img/icons/mediatype/video';
+import { webIcon } from './assets/img/icons/mediatype/web';
+
+@customElement('mediatype-icons')
```

Comment:
Suggestion: Make this singular `mediatype-icon` since its only showing a single icon